cmd/import.go

```diff
@@ -26,7 +26,7 @@ var importCmd = &cobra.Command{
 	},
 	Run: func(cmd *cobra.Command, args []string) {
 		// import only used with additional child commands
-		cmd.Usage()
+		cmd.Help()
```

Comment:
Generally, the Usage term is used in posix/linux. https://en.wikipedia.org/wiki/Usage_message